posts/2020-12-31-Rust-1.49.0.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Announcing Rust 1.49.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.49.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.49.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.49.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1490-2020-12-31
+
+## What's in 1.49.0 stable
+
+For this release, we have some new targets and an improvement to the test
+framework. See the [detailed release notes][notes] to learn about other
+changes not covered by this post.
+
+### 64-bit ARM Linux reaches Tier 1
+
+The Rust compiler supports [a wide variety of targets][platform-support], but
+the Rust Team can't provide the same level of support for all of them. To
+clearly mark how supported each target is we created a tiering system:
+
+* Tier 3 targets are technically supported by the compiler, but we don't check
+  whether their code build or passes the tests, and we don't provide any
+  prebuilt binaries as part of our releases.
+* Tier 2 targets are guaranteed to build and we provide prebuilt binaries, but
+  we don't execute the test suite on those platforms: the produced binaries
+  might not work or might have bugs.
+* Tier 1 targets provide the highest support guarantee, and we run the full
+  suite on those platforms for every change merged in the compiler. Prebuilt
+  binaries are also available.
+
+Rust 1.49.0 promotes the `aarch64-unknown-linux-gnu` target to Tier 1 support,
+bringing our highest guarantees to users of 64-bit ARM systems running Linux!
+We expect this change to benefit workloads spanning from embedded to desktops
+and servers.
+
+This is an important milestone for the project: it's the first time a non-x86
+target reaches the highest level of Quality Assurance and support provided by
```

Comment:
I think my only nit is that we currently still limit crater to x86_64-unknown-linux-gnu; maybe we can just say "It's the first non-x86 target that reaches tier 1"?